packages/cli/src/package.ts

```diff
@@ -35,138 +37,237 @@ export class Package extends Item {
       this.name
     );
   }
-  static async create(directory: string): Promise<Package> {
+  static async create(directory: string, project: Project): Promise<Package> {
     let filePath = nodePath.join(directory, "package.json");
 
     let contents = await fs.readFile(filePath, "utf-8");
     let pkg = new Package(filePath, contents);
+    pkg.project = project;
```

Comment:
could `project` be passed to `Package` constructor to avoid assigning this after creation?

Reply:
Yeah, probably, was just trying to do this quickly to try it out in a project